Version 16.6 is not installed

Compilation error occurs when upgrading from 16.5 to 16.6. I am attaching the installation log. Installed via [pi-apps](https://github.com/Botspot/pi-apps/blob/master/apps/DDNet/install).

**log >>** [install-fail-DDNet.log](https://github.com/ddnet/ddnet/files/10241024/install-fail-DDNet.log)


OS: Raspbian GNU/Linux 10 (buster)
OS architecture: 32-bit
Kernel: armv7l 5.10.103-v7l+
Device model: Raspberry Pi 4 Model B Rev 1.2 
Cpu name: Cortex-A72
Ram size: 3.65 GB
Raspberry Pi OS image version: 2021-05-07
Language: en_US.UTF-8

probs relevant part
```
  Compiling cxxbridge-macro v1.0.71
error[E0658]: `if` is not allowed in a `const fn`
   --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/cxxbridge-macro-1.0.71/src/syntax/discriminant.rs:169:19
    |
169 |               sign: if i < 0 {
    |  ___________________^
170 | |                 Sign::Negative
171 | |             } else {
172 | |                 Sign::Positive
173 | |             },
    | |_____________^
    |
    = note: for more information, see https://github.com/rust-lang/rust/issues/49146

error[E0277]: the trait bound `K: std::cmp::Eq` is not satisfied
  --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/cxxbridge-macro-1.0.71/src/syntax/map.rs:95:26
   |
93 |     impl<K, V> UnorderedMap<K, V> {
   |          - help: consider restricting this bound: `K: std::cmp::Eq`
94 |         pub fn new() -> Self {
95 |             UnorderedMap(HashMap::new())
   |                          ^^^^^^^^^^^^^^ the trait `std::cmp::Eq` is not implemented for `K`
   |
   = note: required by `std::collections::HashMap::<K, V>::new`

error[E0277]: the trait bound `K: std::hash::Hash` is not satisfied
  --> /home/pi/.cargo/registry/src/github.com-1ecc6299db9ec823/cxxbridge-macro-1.0.71/src/syntax/map.rs:95:26
   |
93 |     impl<K, V> UnorderedMap<K, V> {
   |          - help: consider restricting this bound: `K: std::hash::Hash`
94 |         pub fn new() -> Self {
95 |             UnorderedMap(HashMap::new())
   |                          ^^^^^^^^^^^^^^ the trait `std::hash::Hash` is not implemented for `K`
   |
   = note: required by `std::collections::HashMap::<K, V>::new`

error: aborting due to 3 previous errors

```



rust version too old?

@heinrich5991 maybe knows more

--- a/.github/workflows/build.yaml
+++ b/.github/workflows/build.yaml
@@ -88,7 +88,8 @@ jobs:
       if: contains(matrix.os, 'macOS')
       run: |
         brew update || true
-        brew install pkg-config sdl2 ffmpeg python3 ninja molten-vk vulkan-headers glslang spirv-tools
+        # --overwrite for:  Target /usr/local/bin/2to3 already exists.
+        brew install --overwrite pkg-config sdl2 ffmpeg python3 ninja molten-vk vulkan-headers glslang spirv-tools
         brew upgrade freetype
         pip3 install dmgbuild
         echo /Library/Frameworks/Python.framework/Versions/3.11/bin >> $GITHUB_PATH

--- a/README.md
+++ b/README.md
@@ -39,6 +39,8 @@ You can install the required libraries on your system, `touch CMakeLists.txt` an
 
 On older distributions like Ubuntu 18.04 don't install `google-mock`, but instead set `-DDOWNLOAD_GTEST=ON` when building to get a more recent gtest/gmock version.
 
+On older distributions `rustc` version might be too old, to get an up-to-date Rust compiler you can use [rustup](https://rustup.rs/) with stable channel instead or try the `rustc-mozilla` package.
+
 Or on CentOS, RedHat and AlmaLinux like this:
 
     sudo yum install cargo cmake ffmpeg-devel freetype-devel gcc gcc-c++ git glew-devel glslang gmock-devel gtest-devel libcurl-devel libnotify-devel libogg-devel libpng-devel libx264-devel make openssl-devel opus-devel opusfile-devel python2 rust SDL2-devel spirv-tools sqlite-devel vulkan-devel wavpack-devel

